content/en/agent/logs/advanced_log_collection.md

```diff
@@ -443,6 +443,37 @@ More examples:
 | 2020-10-27 05:10:49.657  | `\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d{3}` |
 | {"date": "2018-01-02"    | `\{"date": "\d{4}-\d{2}-\d{2}`                |
 
+### Automatic Multi-line aggregation
+With agent 7.32+ `auto_multi_line_detection` can be enabled which allows the agent to detect common multi-line patterns automatically.
+
+`auto_multi_line_detection` can be enabled globally in the `datadog.yaml`
+
+```yaml
+logs_config:
+  auto_multi_line_detection: true
+```
+
+It can also be enabled or disabled (overriding the global config) per log configuration:
+
+```yaml
+logs:
+  auto_multi_line_detection: true
+```
+
+Automatic mutli-line detection uses a list of common regular expressions to attempt to match logs. If the built in list is not sufficient, you can also add custom patterns in the `datadog.yaml
```

Comment:
Should we add a link to the common regular expressions ?

Reply:
(note: formatting problem here)

```suggestion
Automatic mutli-line detection uses a list of common regular expressions to attempt to match logs. If the built in list is not sufficient, you can also add custom patterns in the `datadog.yaml`
```

Also, can extra patterns be defined at the logs integration level as well?

Reply:
> Also, can extra patterns be defined at the logs integration level as well

No because it really doesn't make sense IMO. If you are going to make a log integration you may as well just add a normal multi line pattern. WDYT?